### PR TITLE
#8378 - new Excel commands described

### PR DESCRIPTION
**Pull request by Vegebutcher**
copy, paste, selectrange

### CODE REVIEW

File: G1ANT.Addon.MSOffice/Commands/ExcelCopyCommand.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+```
+
+## Description
+
+This command copies the content of the currently selected cells to the clipboard.
+
+| Argument | Type | Required | Default Value | Description |
+| -------- | ---- | -------- | ------------- | ----------- |
+| `if`           | [bool](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/BooleanStructure.md) | no       | true                                                        | Executes the command only if a specified condition is true   |
+| `timeout`      | [timespan](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/TimeSpanStructure.md) | no       | [♥timeoutcommand](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Addon.Core/Variables/TimeoutCommandVariable.md) | Specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.MSOffice/Commands/ExcelCopyCommand.md]: incorrect link https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Addon.Core/Variables/TimeoutCommandVariable.md

File: G1ANT.Addon.MSOffice/Commands/ExcelPasteCommand.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+```
+
+## Description
+
+This command inserts the clipboard content into the currently selected cell or range.
+
+| Argument | Type | Required | Default Value | Description |
+| -------- | ---- | -------- | ------------- | ----------- |
+| `if`           | [bool](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/BooleanStructure.md) | no       | true                                                        | Executes the command only if a specified condition is true   |
+| `timeout`      | [timespan](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/TimeSpanStructure.md) | no       | [♥timeoutcommand](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Addon.Core/Variables/TimeoutCommandVariable.md) | Specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.MSOffice/Commands/ExcelPasteCommand.md]: same

File: G1ANT.Addon.MSOffice/Commands/ExcelSelectRangeCommand.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+| `colindex2` or `colname2` | [integer](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/IntegerStructure.md)  or [text](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/TextStructure.md) | yes      |                                                              | `colindex2`: ending cell's column number; `colname2`: ending cell's column name |
+|`row2`| [integer](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/IntegerStructure.md) | yes |  | Ending ell's row number |
+| `if`           | [bool](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/BooleanStructure.md) | no       | true                                                        | Executes the command only if a specified condition is true   |
+| `timeout`      | [timespan](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/TimeSpanStructure.md) | no       | [♥timeoutcommand](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Addon.Core/Variables/TimeoutCommandVariable.md) | Specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.MSOffice/Commands/ExcelSelectRangeCommand.md]: same

**Review by WiktoriaPrusik — CHANGES_REQUESTED** [review]

File: G1ANT.Addon.MSOffice/Commands/ExcelCopyCommand.md (hunk 1 above)
Comment (Vegebutcher) [reply G1ANT.Addon.MSOffice/Commands/ExcelCopyCommand.md]:
```suggestion
| `timeout`      | [timespan](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/TimeSpanStructure.md) | no       | [♥timeoutcommand](https://manual.g1ant.com/link/G1ANT.Addon.Core/G1ANT.Addon.Core/Variables/TimeoutCommandVariable.md) | Specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
```

File: G1ANT.Addon.MSOffice/Commands/ExcelPasteCommand.md (hunk 2 above)
Comment (Vegebutcher) [reply G1ANT.Addon.MSOffice/Commands/ExcelPasteCommand.md]:
```suggestion
| `timeout`      | [timespan](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/TimeSpanStructure.md) | no       | [♥timeoutcommand](https://manual.g1ant.com/link/G1ANT.Addon.Core/G1ANT.Addon.Core/Variables/TimeoutCommandVariable.md) | Specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
```

File: G1ANT.Addon.MSOffice/Commands/ExcelSelectRangeCommand.md (hunk 3 above)
Comment (Vegebutcher) [reply G1ANT.Addon.MSOffice/Commands/ExcelSelectRangeCommand.md]:
```suggestion
| `timeout`      | [timespan](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/TimeSpanStructure.md) | no       | [♥timeoutcommand](https://manual.g1ant.com/link/G1ANT.Addon.Core/G1ANT.Addon.Core/Variables/TimeoutCommandVariable.md) | Specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
```

**Review by WiktoriaPrusik — APPROVED** [review]

**Pull request merged**